### PR TITLE
LPFG-1011 make ambiguous links more meaningful

### PR DESCRIPTION
**Pull request by mick-jones**
- added (hidden) course title to add/remove links in order for them to be more meaningful to screen readers
- made the hidden text for change primary area of work/ change other areas of work unique so that screen readers can tell them apart

### CODE REVIEW

File: src/ui/component/Nav.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <nav id="proposition-menu">
 
     <a href="/" title="Go to the Learning Platform homepage" id="proposition-name">
-        Civil Service Learning
+        Civil Service Learning
```
Comment (elxsy) [inline src/ui/component/Nav.html]: extra space mistake?

File: src/ui/component/disciteParts/DisciteCourseProperties.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
         <li class="discite__property-item">
             <span class="visuallyhidden">Duration:</span> {course.getDuration() || ''}
         </li>
-        {#if !course.isRequired()}
+        {#if !course.isRequired()}
```
Comment (elxsy) [inline src/ui/component/disciteParts/DisciteCourseProperties.html]: extra space?

File: src/ui/component/Nav.html (hunk 1 above)
Comment (mick-jones) [reply src/ui/component/Nav.html]: removed

File: src/ui/component/disciteParts/DisciteCourseProperties.html (hunk 2 above)
Comment (mick-jones) [reply src/ui/component/disciteParts/DisciteCourseProperties.html]: Removed

**Pull request merged**